### PR TITLE
Allows pg_state gauges to return to 0

### PR DESCRIPTION
**Pull request by nickjanus**
This does change the default behaviour for these metrics. Previously, these time series did not exist until ceph_exporter observed a pg in the given state. Now these time series will always exist and be zero whenever ceph_exporter sees no pgs in the given state.

Also run `go fmt`, because we're not animals. :heart:

### CODE REVIEW

**Comment by nickjanus** [comment]
WIP

File: collectors/health.go [hunk 1]
```diff
@@ -1206,76 +1206,40 @@ func (c *ClusterHealthCollector) collect(ch chan<- prometheus.Metric) error {
...
-	if *pgStateMap["down"] > 0 {
-		c.DownPGs.Set(*pgStateMap["down"])
-		c.PGState.WithLabelValues("down").Set(*pgStateMap["down"])
+	for state, gauge := range pgStateGaugeMap {
+		val := *pgStateCounterMap[state]
+		if state == "scrubbing" {
+			val -= *pgStateCounterMap["scrubbing+deep"]
+		}
+		gauge.Set(val)
+		c.PGState.WithLabelValues(val).Set(val)
```
Comment (neurodrone) [inline collectors/health.go:1242]: `WithLabelValues(state)` ?

File: collectors/health.go [hunk 2]
```diff
@@ -1206,76 +1206,40 @@ func (c *ClusterHealthCollector) collect(ch chan<- prometheus.Metric) error {
...
-		for pgState := range pgStateMap {
+		for pgState := range pgStateCounterMap {
 			if strings.Contains(p.States, pgState) {
-				*pgStateMap[pgState] += p.Count
+				*pgStateCounterMap[pgState] += p.Count
 			}
 		}
 	}
 
-	if *pgStateMap["degraded"] > 0 {
```
Comment (baergj) [inline collectors/health.go:1219]: It makes me so happy that you got rid of this line of `if` statements!

Comment (nickjanus) [reply collectors/health.go:1219]: A combination of golang and the prom library actually makes this inordinately difficult, since `prometheus.Gauge` is actually an interface, and you can't have pointers to an interface type, so back to the drawing board...

Comment (nickjanus) [reply collectors/health.go:1219]: Derp, I got confused by some other test output when I removed the `&` from the map literal. I suppose it works because `c` is a pointer receiver. I'm always a little surprised by how nicely references work in golang.

File: collectors/health_test.go [hunk 3]
```diff
@@ -31,23 +31,6 @@ func TestClusterHealthCollector(t *testing.T) {
 	}{
 		{
 			input: `
-{
```
Comment (neurodrone) [inline collectors/health_test.go:34]: Why did we need to take these out?

Comment (nickjanus) [reply collectors/health_test.go:34]: These test cases failed becase the logic for setting the gauges based on `pgmap` would set all the pg state related gauges to 0, failing the tests. If `pgmap` is always present, then we're good. If there's an edge case where only summary is present, then I need to adjust this approach.

Comment (nickjanus) [reply collectors/health_test.go:34]: To be clear, I've tested this out and it seems to work fine. Just wanted to make sure that the exporter wasn't setting those gauges redundantly for a reason.

Comment (neurodrone) [reply collectors/health_test.go:34]: Gotcha. Sounds good to me!
